### ISSUE TITLE
[DESIGN] Form input borders have insufficient contrast (WCAG 1.4.11 AA)

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#412]
### Summary

Refer to: https://github.com/civictheme/uikit/issues/372.

Within the contact form, the contrast between the light blue (#e6e9eb) input borders and the adjacent white (#ffffff) background does not meet the minimum ratio required.

### Steps to reproduce

Inspect colour of input borders on form fields.

### Observed outcome

Within the contact form, the contrast between the light blue (#e6e9eb) input borders and the adjacent white (#ffffff) background does not meet the minimum ratio required.

• Current contrast ratio: 1.21:1

### Expected outcome

• Required contrast ratio: 3:1
Form input field borders need to be WCAG 2.2 accessible.

**Comment by fionamorrison23** [comment]
Closing this issue as it's a duplicate of https://github.com/civictheme/uikit/issues/372
